up-linux-streamer/src/main.rs

```diff
@@ -46,9 +52,20 @@ async fn main() -> Result<(), UStatus> {
     let mut streamer = UStreamer::new(
         "up-linux-streamer",
         config.up_streamer_config.message_queue_size,
+        usubscription,
     );
 
-    let zenoh_config = ZenohConfig::default();
+    let mut zenoh_config = ZenohConfig::default();
+
+    // Specify the address to listen on using IPv4
+    let ipv4_endpoint = ZenohEndpoint::from_str("tcp/0.0.0.0:7447");
+
+    // Add the IPv4 endpoint to the Zenoh configuration
+    zenoh_config
```

Comment:
IMHO any Zenoh configuration we need to do should be possible at a minimum via configuration file, not baked directly in here.

You can accomplish this via modifying config.rs in order to add the necessary field to the appropriate spot.